spec/features/api_access_spec.rb

```diff
@@ -9,18 +9,25 @@ def sign_in_as_admin
                       "admin[password]" => @admin.password)
   end
 
-  def response_json
-    JSON.parse( response.body )
+  before(:each) do
+    sign_in_as_admin
   end
 
   describe "api json export" do
-    it 'routes to the all action on default' do
+    it 'exports all organizations as json by default' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.